src/features/list-view/__tests__/ListView-test.js

```diff
@@ -1,9 +1,28 @@
+// @flow
 import * as React from 'react';
 
 import ListView from '../ListView';
-
 // Global Declarations
-const rowData = [['', 'A', 'B', 'C'], ['', 'D', 'E', 'F']];
+
+const nameCellStyle = { height: 50, left: 0, position: 'absolute', top: 0, width: 300 };
+
+type Props = {|
+    className: string,
+    key: string,
+    name: string,
+|};
+
+const ListViewNameTestCell = ({ name }: Props) => <div>{name}</div>;
```

Comment:
Nit: Maybe we can do away with the 'Test' in the name. Since this component is being used in a test file I think it's safe to assume that these component are test components.

ListViewNameTestCell -> ListViewNameCell

Reply:
Same goes for the `testCell`, we can probably call that `Cell` instead.